.github/workflows/node.js.yml

```diff
@@ -12,18 +12,40 @@ on:
 jobs:
   build:
 
-    runs-on: ubuntu-latest #Can change the runtime environment according to needs
+    runs-on: ${{ matrix.os }}
 
     strategy:
       matrix:
         node-version: [14.x, 16.x]
         # See supported Node.js release schedule at https://nodejs.org/en/about/releases/
+        os: [ubuntu-latest, macOS-latest, windows-latest]
```

Comment:
this will eat into our free github action minutes. just ubuntu is fine.

Reply:
As you mentioned it is free, so why not use it 😄

Reply:
Only 2000 minutes are free. Not unlimited :(

Reply:
I thought 2000 minutes are free for private repo.

But I agree with your point on carbon footprint but unless we see any raised issue about the same let us stick with ubuntu